component-generator/utils.js

```diff
@@ -1,7 +1,9 @@
 const { InvalidOptionArgumentError } = require('commander');
 const fs = require('fs');
 const path = require('path');
+const axios = require('axios');
 const { exec } = require('child_process');
+require('dotenv').config({ path: path.resolve(__dirname, '../www/.env.development') });
```

Comment:
The `BASE_URL` and `TRACK_ANONYMOUS_ANALYTICS` are defined in `www/.env.development` but they are only related to the component generator. Should these environment variables be moved to a .env file in `component-generator` directory?

Reply:
Yes, it is moved